src/test/kotlin/app/cash/licensee/LicenseePluginFixtureTest.kt

```diff
@@ -280,18 +289,23 @@ class LicenseePluginFixtureTest {
     assertThat(result.output).containsMatch(
       """
       |WARNING: Allowed .*? is unused
-      """.trimMargin(),
+      """.trimMargin().toRegex(),
     )
   }
 
   @Test fun allFixturesCovered() {
-    val expectedDirs = javaClass.declaredMethods
+    val expectedDirs = javaClass.declaredMethods.asSequence()
       .filter { it.isAnnotationPresent(Test::class.java) }
       .filter { it.parameterCount == 1 } // Assume single parameter means test parameter.
       .flatMap { it.parameters[0].getAnnotation(TestParameter::class.java).value.toList() }
-      .toSet()
-    val actualDirs = fixturesDir.listFiles()!!.filter { it.isDirectory }.map { it.name }
-    assertThat(expectedDirs).containsExactlyElementsIn(actualDirs)
+      .sorted()
+    val actualDirs = fixturesDir.listFiles()!!.asSequence()
+      .filter { it.isDirectory }
+      .map { it.name }
+      .sorted()
+      .toList()
+      .toTypedArray()
+    assertThat(expectedDirs).containsExactly(*actualDirs)
```

Comment:
Expect/actual needs flipped. This one was backwards before, but since you're already in here...